### PR TITLE
Remove the admin UI

### LINKED ISSUE #8273: Remove the old admin UI
Remove all the javascript and everything else from the old admin UI. Have the spot in the config file that used to have the port location with a commented line that tells them the new admin UI is in the Chronograf project here: https://github.com/influxdata/chronograf

### PR DESCRIPTION
**Pull request by jsternberg**
Fixes #8273.

- [x] Rebased/mergable
- [x] Tests pass
- [x] CHANGELOG.md updated
- [x] Config changes: update sample config (`etc/config.sample.toml`), server `NewDemoConfig` method, and `Diagnostics` methods reporting config settings, if necessary
- [x] [InfluxData Documentation](https://github.com/influxdata/docs.influxdata.com): issue filed or pull request submitted \<[influxdata/docs.influxdata.com#1107](https://github.com/influxdata/docs.influxdata.com/issues/1107)\>

### CODE REVIEW

File: Godeps [hunk 1]
```diff
@@ -3,7 +3,6 @@ github.com/BurntSushi/toml 99064174e013895bbd9b025c31100bd1d9b590ca
 github.com/bmizerany/pat c068ca2f0aacee5ac3681d68e4d0a003b7d1fd2c
 github.com/boltdb/bolt 4b1ebc1869ad66568b313d0dc410e2be72670dda
 github.com/cespare/xxhash 4a94f899c20bc44d4f5f807cb14529e72aca99d6
-github.com/clarkduvall/hyperloglog 2d38f733946d0a1f2e810513c71b834cbeba1480
```
Comment (jwilder) [inline Godeps]: @e-dard Is this used by TSI?

Comment (jsternberg) [inline Godeps]: I couldn't figure out where this came from. `statik` doesn't seem to use it, but gdm also didn't pick it up.

Comment (jsternberg) [inline Godeps]: I'll add it back. It looks like it's used by some package in `pkg` which is used by TSI. I have no idea why gdm couldn't find it.

Comment (jsternberg) [inline Godeps]: It seems like an earlier version of TSI might have used this. There's a package in the `pkg` folder named `hll` that references `hyperloglog` and says it's a faster version. I can't find any direct references to `hyperloglog` though.

Comment (e-dard) [inline Godeps]: @jwilder @jsternberg yeah I think I meant to remove this. It can go.

**Pull request merged**